### ISSUE TITLE
Training results and documents are somewhat different

### ISSUE THREAD

**Issue by linzihyuan** [AngusG/tensorflow-xnor-bnn#13]
Hi

I'm the student from Chung Cheng Univ. in Taiwan who are interested in your Source code about BNN. 

After reading the "readme" file, I was trying to reproduce your experiment results.

However, the results are not as expected.

The experimental accuracy is only 88% and does not reach 96% mentioned in the "readme" file.

Could you please give me some directions? Is this a reasonable result?

Thanks!

![gpu_ver](https://user-images.githubusercontent.com/43560196/53234687-1203a300-36cb-11e9-95f2-edcdf9176275.png)

![default](https://user-images.githubusercontent.com/43560196/53234721-2051bf00-36cb-11e9-8b1c-c6fdb7e8a7ed.PNG)

**Comment by AngusG** [comment]
Hi, I'm not actively supporting this repo at the present time, but I can offer some suggestions. 

It's possible the default initialization scheme has changed since I last worked on this (TF version 1.2) which could account for the discrepancy you are seeing with the readme (assuming everything else is the same). 

There will also be some sensitivity to the random seed, which you need to set twice in tensorflow for completely deterministic results, once at the graph level and in `get_variable`, or similar (I did neither here). The difference in loss at initialization (5.7 vs 2.3) does seem a bit concerning.

In any case, I set the default number of steps to run quickly on a basic GPU, not train a model to convergence, so if you really want to see high accuracy I suggest bumping up the number of epochs.

**Comment by linzihyuan** [comment]
Thank you for your reply. I got it

**Comment by akm190** [comment]
Hi @linzihyuan 
I am also seeing the same difference in the results. 
Did you figure out this problem?
